requirements.txt

```diff
@@ -26,7 +26,7 @@ mkdocstrings==0.20.0
 mkdocstrings-python==0.8.3
```

Comment:
Just to confirm, are these versions compatible with the ones listed under `setup.py`?